llama_hub/library.json

```diff
@@ -252,6 +252,11 @@
     "author": "sidu",
     "keywords": ["jokes", "dad jokes"]
   },
+  "WordLiftLoader": {
+    "id": "WordLift",
```

Comment:
nit: should be same as folder name. can you change this to `"id": "wordlift"`? thanks

Reply:
@jerryjliu I changed to `"id": "wordlift"`. Thanks.